mobile/graphics.html

```diff
@@ -43,6 +43,7 @@ <h2>Technologies in progress</h2>
 
         <div data-feature="Downloadable fonts">
           <p>Given the time required for downloading fonts over mobile networks, authors need to adapt their content to the progressive availability of fonts; <a data-featureid="css-font-loading">CSS Font Loading</a> gives the necessary events to developers to enable that adaptation.</p>
+          <p>Font variations along different axes (width, weight, optical size) can be encoded in a single font file. The <a data-featureid="css-fonts-variable">Variable fonts properties</a> in CSS allow finer control over usage of font variants and can further help developers avoid unnecessary network requests to fetch additional variants of a font.</p>
```

Comment:
s/Variable fonts properties/font variation properties/ to use the term in the spec.